internal/commands/vaultsecrets/secrets/secrets.go

```diff
@@ -34,6 +37,13 @@ func NewCmdSecrets(ctx *cmd.Context) *cmd.Command {
 		},
 	}
 
+	// Autocomplete the persistent flag.
+	for _, f := range cmd.Flags.Persistent {
```

Comment:
Similar question here. This was working with inline autocomplete. Can't we inline this as well?

Reply:
Same answer as above. You need to parse the flags in the autocomplete function and thus you need access to the command.